src/sniffer/linux/IOHandlerHw.c

```diff
@@ -0,0 +1,86 @@
+
+#include "IOHandler.h"
+
+#include <sys/types.h>
+#include <signal.h>
+#include <fcntl.h>
+#include <string.h>
+#include <unistd.h>
+
+#include "ErrorHandler.h"
+
+#define PIPE_READ_END 0
+#define PIPE_WRITE_END 1
+
+int IOHandler_loggerStdInWr;
+pid_t IOHandler_loggerProcessId;
+
+/* reverse:  reverse string s in place */
+static void reverse(char s[])
```

Comment:
I think a new file should be created for `reverse` and `itoa` (and the ones to come).

Reply:
Yes we could create a utils or something similar, but as they are only for linux I left them here for now.

Reply:
Okay, let's merge this one as it is and we'll take care of this later.